### ISSUE TITLE
Multi-device BETA feature

### ISSUE THREAD

**Issue by arrichter** [tomer8007/whatsapp-web-incognito#46]
Hi everyone,
as [mentioned](https://github.com/tomer8007/whatsapp-web-incognito/issues/44#issuecomment-882090951) by [tomer8007](https://github.com/tomer8007), there is currently no support for the new Multi-device BETA feature.
So far there was no issue listed with respect to this, so here we are.
Best,
Alex

**Comment by zakibaihaki23** [comment]
when the extension get update for multi device feature? 
thanks

**Comment by tomer8007** [comment]
This will take some time as this feature is using the [Noise Protocol](https://noiseprotocol.org/) and [Signal](https://en.wikipedia.org/wiki/Signal_Protocol), as the iOS/Android clients do.

**Comment by 9Datenshi** [comment]
Thank you for your work and dedication to this project!

I noticed that there are still some bugs in the multi-device version, it seems that contacts are working perfectly. But the groups still have errors when viewing, even after pressing "read" the message appears as unread. And even after sending messages the group appears as unread.
I also noticed that it happens even activating the timing, it is marked as read, but it goes back to the messages as if they were not read.
